deepgrp/__main__.py

```diff
@@ -310,16 +310,43 @@ def train(args: argparse.Namespace, options: dgmodel.Options) -> None:
         parameter.fromdict(options.todict())
         logdir = args.logdir
 
-        # add _LOG.xxx functions for logging
-        # include a check if preprocessed?
+        # get which chromosome is used for training and which for validation
+        if args.trainfile[-10:] != '.fa.gz.npz' or \
+            args.validfile[-10:] != '.fa.gz.npz':
+                # train or valid file is not valid
+                sys.stderr.write('Training and validation file given must be ' +
+                                'in the format chr[chrnumber].fa.gz.npz, ' +
+                                'e.g. chr1.fa.gz.npz\n')
```

Comment:
The file format should have `.npz`, if there is `.fa.gz` is before should not matter. 
We don't need to be too restrictive on the file name. 
I think it is fine if they are in the format `something[optional_additional_suffixes].npz` and I don't think we need to check for this. So you can remove this block.